google/cloud/internal/async_streaming_write_rpc_tracing.h

```diff
@@ -36,11 +36,7 @@ class AsyncStreamingWriteRpcTracing
       : context_(std::move(context)),
         impl_(std::move(impl)),
         span_(std::move(span)) {}
-  ~AsyncStreamingWriteRpcTracing() override {
-    if (context_) {
-      (void)EndSpan(*std::move(context_), *std::move(span_), Status());
-    }
-  }
+  ~AsyncStreamingWriteRpcTracing() override { (void)End(StatusOr<Response>()); }
```

Comment:
The default constructed `StatusOr` is not an `OK` status, but an `UNKNOWN` error:

https://github.com/googleapis/google-cloud-cpp/blob/c3f7035ac1700b0635d5c593cc9efbc10fb1e935/google/cloud/status_or.h#L101-L104

While I think either status code is a reasonable representation of what is happening, I would prefer to leave the behavior unchanged, and report `OK`. 

We can do that with: `make_status_or<Response>({})`.

Reply:
Done.